src/panel/app_panel.js

```diff
@@ -35,6 +35,15 @@ export default class App {
       </DeviceProvider>,
       document.querySelector('#react_root')
     );
+
+    // The mouseLeave flag allows to keep the suggest open when clicking outside of the browser
+    window.mouseLeave = false;
+    window.document.onmouseleave = () => {
+      window.mouseLeave = true;
+    };
+    window.document.onmouseenter = () => {
+      window.mouseLeave = false;
+    };
```

Comment:
On my computer it only appears to work on Chromium but no on Firefox.

This could be specific to i3 which handle window focus slightly differently, does it work on other setups?

Reply:
nice catch, fixed!